lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -45,10 +49,23 @@ func Init(portLayerAddr string) error {
 	portLayerServerAddr = portLayerAddr
 
 	imageCache = cache.NewImageCache()
-	// update the image cache at startup
-	if err := imageCache.Update(portLayerClient); err != nil {
-		return fmt.Errorf("Error refreshing image cache: %s", err)
-	}
+
+	// attempt to update the image cache at startup
+	log.Info("Refreshing image cache...")
+	go func() {
+		for i := 0; i < Retries; i++ {
+
+			// initial pause to wait for the portlayer to come up
+			time.Sleep(RetryTimeSeconds * time.Second)
+
+			if err := imageCache.Update(portLayerClient); err == nil {
+				log.Info("Image cache updated successfully")
+				return
+			}
+			log.Info("Failed to refresh image cache, retrying...")
+		}
+		log.Warn("Failed to refresh image cache. Is the portlayer server down?")
+	}()
 
 	return nil
```

Comment:
Not very.. the persona would have no knowledge of what lives on the portlayer. I've refactored the `Images` function to rely on the cache. I considered going around the cache directly to the portlayer in the event that the cache wasn't updated, but if the cache isn't updated, it's because the portlayer is either down or unreachable, which should demand some attention.